### DISCUSSION TITLE
WAL2JSON Examples

### DISCUSSION THREAD

**Discussion by SeanDuHare** [eulerto/wal2json#231] [Q&A] [answered]
Hello,

Are there examples of the output following this plug-in somewhere? Specifically, I'd like to see the output when this is used to output logs from a table that has foreign key constraints. I am looking into this to answer the following questions:
* Can we dynamically determine if a column is a foreign key
* Can we determine what schema & table this foreign key relates to?
* Do we, or can we, get the row the foreign key refers to?

**Comment by eulerto** [comment, marked as answer]
There are some basic examples available in the [README](https://github.com/eulerto/wal2json/blob/master/README.md). wal2json does not provide foreign key information. What is the use case?
